Field.h

```diff
@@ -125,4 +126,38 @@ class TextField: public Field {
         string getValue() {
             return fieldValue;
         }
+};
+
+class CharField: public Field {
+    
+    private:
+        string fieldValue;
+        int maxLength;
+        
+    public:
+        CharField(string name, int maxLen, string defaultValue="", bool primary=false, bool unique=false, bool nullable=true):
```

Comment:
Second parameter should be `maxLength`. I prefer using the full word e.g. `defaultValue` instead of `defaultVal`. Keeping these sorts of things consistent lets people guess correctly.